README.md

```diff
@@ -23,11 +24,14 @@ Gitcoin pushes Open Source Forward. Learn more at [https://gitcoin.co](https://g
 Built as part of the [ETHWaterloo Hackathon 2017](https://ethwaterloo.com).
 
 
+# Credits
 
-# credits
+Thanks to  [Koloda](https://github.com/Yalantis/Koloda).
 
-Thanks to Richard Kim for [RKSwipeCards](https://github.com/cwRichardKim/RKSwipeCards).
 
+# Setup
+1. Install cocoapods, and run pod install
+2. Create a SafeConfiguration.plist file and add the following keys as strings: gitHubOAuthToken, gitHubOAuthSecret
```

Comment:
is it worth creating a `SafeConfiguration.plist.dist` file as a template?

Reply:
Maybe? I am not familiar with the *.dist pattern...

the Xcode project will have the missing file -> https://screencast.com/t/fQ1R2bMbnC
and i suppose if this file gets larger we will need something

perhaps a script that generates the file or something?

Reply:
the *.dist pattern is just a sample file with fake keys in it.

Reply:
id say we just stick the sample in the readme... for now

Reply:
fixed in c7fc847